skimage/registration/_tvl1.py

```diff
@@ -0,0 +1,206 @@
+# coding: utf-8
+"""TV-L1 optical flow algorithm implementation.
+
+"""
+
+from functools import partial
+import numpy as np
+from scipy import ndimage as ndi
+from skimage.transform import warp
+
+from .utils import coarse_to_fine
+
+
+def _tvl1(I0, I1, flow0, dt, lambda_, tau, nwarp, niter, tol, prefilter):
```

Comment:
As mentioned below, use `*` after `image1` to make later arguments keyword-only.

Reply:
`_tvl1` is supposed to be hidden and only called internally as a routine, so I think that there is no need for such precautions.